packages/celotool/src/lib/oracle.ts

```diff
@@ -0,0 +1,30 @@
+import { getFornoUrl } from 'src/lib/endpoints'
+import { envVar, fetchEnv } from 'src/lib/env-utils'
+import { installGenericHelmChart } from 'src/lib/helm_deploy'
+
+const helmChartPath = '../helm-charts/oracle'
+
+export async function installHelmChart(celoEnv: string) {
+  return installGenericHelmChart(
+    celoEnv,
+    releaseName(celoEnv),
+    helmChartPath,
+    helmParameters(celoEnv)
+  )
+}
+
+function helmParameters(celoEnv: string) {
+  return [
+    `--set celotool.image.repository=${fetchEnv('CELOTOOL_DOCKER_IMAGE_REPOSITORY')}`,
```

Comment:
nit: some of these string literals are already found in `envVar`, if they aren't we should add them and use them to avoid hardcoding strings